### PR TITLE
Add teams section

### LINKED ISSUE #2: Add teams section to get members of a team
# Summary

Add teams section to get members of a team.

# Changes

I guess `settings.yml` format will be the following.

```yml
users:
    - id: user1
    - id: user2
teams:
    - id: team1
    - id: team2
```

And it would be able to get members to call API.
https://developer.github.com/v3/teams/members/#list-team-members

### PR DESCRIPTION
**Pull request by kyoshidajp**
Fix #2 

And use an environment variable `GITHUB_TOKEN` to call Github API if this specified.

### CODE REVIEW

**Comment by potch8228** [comment]
Did you run `go test` ?

**Comment by kyoshidajp** [comment]
No. I changed tests. However, tests still fail.

The reason is some user doesn't have a pubkey. So, `TestPubKey_LoadPubKeys` and `TestPubKey_OutputList` will fail. WDYT?

**Comment by potch8228** [comment]
The purpose of this tool is getting SSH public keys for authorized_keys file.
I know that GitHub let you submit GPG key, but that's not what we want for this tool.

The answer is just ignore it... but I think it will be nicer to tell on stderr that non-SSH public key is detected and ignored it.

...BUT, is that really not SSH public key??

File: pubkey.go [hunk 1]
```diff
@@ -45,26 +59,68 @@ func (p *PubKey) loadFile(filename string) {
...
+			p.users[users[i]] = u
+		}
+	}
+}
+
+func (p *PubKey) GetMembers(team string) []string {
+	users, resp, err := p.client.Organizations.ListMembers(context.Background(), team, nil)
+	if err != nil {
+		log.Println(resp)
+		log.Fatalln("Failed to fetch members: " + team)
```
Comment (potch8228) [inline pubkey.go:86]: Just in case to make sure, `log.Fatalln` or `log.Fatal**` will cause panic; did you intend?

**Review by potch8228 — CHANGES_REQUESTED** [review]

File: pubkey.go (hunk 1 above)
Comment (potch8228) [reply pubkey.go:86]: no... `log.Fatal**` triggers `os.Exit(1)`... which anyways stops program.

Comment (kyoshidajp) [reply pubkey.go:86]: Oh, I didn't know that. I'm going to replace `log.Fatalln` with `log.Println` to continue processes.

**Review by potch8228 — APPROVED** [review]
LGTM

But one last thing, gotta remove the binary outputs :)

**Comment by potch8228** [comment]
Thanks for your works anyways though!

**Pull request merged**